tests/ResourceTests.cpp

```diff
@@ -821,10 +851,10 @@ TEST_CASE("ComTokenDirectCloser", "[resource]")
 
 TEST_CASE("UniqueCloseClipboardCall", "[resource]")
 {
-#if defined(__WIL__WINUSER_) && !defined(NOCLIPBOARD)    
+#if defined(__WIL__WINUSER_) && !defined(NOCLIPBOARD)
```

Comment:
unrelated -- may want to back this out?

Reply:
Yeah I wasn't sure if I should keep it. Other files all have a trailing newline and stripped whitespace, so I kept it for my initial submission, but I can definitely remove it if anyone would like to. (Unless you meant just that.)

Reply:
We talked outside of GitHub, and I've decided to keep this change at the moment until anyone else chimes in.